tests/models_tests/models_test.go

```diff
@@ -814,17 +814,22 @@ func TestRetrieveDataBySourceIdAndCohortIdAndConceptIdsOrderedByPersonId(t *test
 			t.Errorf("Expected some cohort data")
 		}
 		var previousPersonId int64 = -1
-		defaultHareValue := map[string]bool{"": true, "non-Hispanic Asian": true, "non-Hispanic Black": true, "non-Hispanic White": true, "Hispanic": true}
-
+		defaultHareValue := map[string]bool{"non-Hispanic Asian": true, "non-Hispanic Black": true, "non-Hispanic White": true, "Hispanic": true}
+		emptyDataCounter := 0
```

Comment:
While testing locally I noticed cohortData exploded. I think the query might be wrong. Please check.
Please also add this check to complement the `// 1- ` check above and guard against regression:
```go
		// 2- check data size. Max size, if all persons have data for each concept, is cohort size x len(allConceptIds):
		if len(cohortData) > cohortDefinition.CohortSize*len(allConceptIds) {
			t.Errorf("Cohort data size does not match expectation. Expected < %d, got %d",
				cohortDefinition.CohortSize*len(allConceptIds), len(cohortData))
		}
```

Reply:
issue could not be reproduced. But test/check added to make sure.